.babelrc

```diff
@@ -1,3 +1,4 @@
+// Only use to transpile JSX from app/javscript/src/Stats
```

Comment:
I've been tempted of removing this file several times, since it is not used apparently. Anyway, it's not used anymore by webpacker and it's gonna be removed sooner than later in favor of `babel.config.js`. I wouldn't sweat about it.